.travis.yml

```diff
@@ -124,6 +124,7 @@ env:
     - TRAVIS_FLAVOR=twemproxy FLAVOR_VERSION=latest
     - TRAVIS_FLAVOR=varnish FLAVOR_VERSION=4.1.7
     - TRAVIS_FLAVOR=varnish FLAVOR_VERSION=5.0.0
+    - TRAVIS_FLAVOR=squid FLAVOR_VERSION=latest
```

Comment:
Can we pin to some fixed version? Otherwise when a new version is released the tests might fail unexpectedly.

Reply:
I used the container `datadog/squid` available here: https://hub.docker.com/r/datadog/squid/.
The only tag available is `latest`. I don't know who's responsible for this one, maybe we could tag the release.

Reply:
Let's tag it - we can merge this as is, but we should pin that once we tag it properly.